data/mods/Magiclysm/enchanted/enchanted_melee.json

```diff
@@ -754,5 +754,176 @@
     "name": "dao +2",
     "proportional": { "price": 6.0, "bashing": 1.2, "cutting": 1.2, "weight": 0.8 },
     "relative": { "to_hit": 2 }
+  },
+  {
+    "id": "rune_biomancer_weapon",
+    "type": "GENERIC",
+    "name": "Biomancer spear",
+    "plural": "Biomancer spears",
+    "\\": "Effectively a durable stone spear +1 with IMPALE tech. based off Wicked Bonespear. Lorewise, it would make sense for novice Biomancers to be developing their skills in spears so they can adequately use the Bonespear(No whips/REACH3 for you!) once they're proficient enough to summon one. Should be weaker than other school weapons DPSwise due to reach.",
+    "description": "A grotesque bone spearhead on a stout wooden pole. There is a Biomancer rune embedded at the base of the head.",
+    "weight": 988,
+    "volume": "1250 ml",
+    "price": 3900,
+    "to_hit": 2,
+    "bashing": 6,
+    "cutting": 15,
+    "material": [ "wood", "bone" ],
+    "symbol": "/",
+    "color": "light_gray",
+    "techniques": [ "WBLOCK_1", "IMPALE" ],
+    "qualities": [ [ "COOK", 1 ] ],
+    "flags": [ "SPEAR", "REACH_ATTACK", "DURABLE_MELEE", "SHEATH_SPEAR", "TRADER_AVOID", "MAGIC_FOCUS" ]
+  },
+  {
+    "id": "rune_technomancer_weapon",
+    "type": "GENERIC",
+    "name": "Technomancer toolbar",
+    "plural": "Technomancer toolbars",
+    "\\": "Combat stats based off crowbar +1, techniques off Halligan bar(closest non-magical substitute), weight based on 75% of hammer & wrench(since their handles aren't included) + 100% crowbar with benefit from +1 enchantment",
```

Comment:
```suggestion
    "//": "Combat stats based off crowbar +1, techniques off Halligan bar(closest non-magical substitute), weight based on 75% of hammer & wrench(since their handles aren't included) + 100% crowbar with benefit from +1 enchantment",
```